apiserver/Makefile

```diff
@@ -97,12 +97,12 @@ sub-image-fips-%:
 
 $(API_SERVER_IMAGE): $(CONTAINER_MARKER)
 $(CONTAINER_CREATED): Dockerfile $(BINDIR)/apiserver-$(ARCH)
-	$(DOCKER_BUILD) --build-arg BIN_DIR=$(BINDIR) -t $(API_SERVER_IMAGE):latest-$(ARCH) -f Dockerfile .
+	$(DOCKER_BUILD) --build-arg BIN_DIR=$(BINDIR) --build-arg CALICO_BASE=$(CALICO_BASE) -t $(API_SERVER_IMAGE):latest-$(ARCH) -f Dockerfile .
```

Comment:
You can add the `CALICO_BASE` arg to [`DOCKER_BUILD`](https://github.com/projectcalico/calico/blob/master/lib.Makefile#L267) so that you don't need to change every Makefile.